src/controller/python/ChipDeviceController-ScriptBinding.cpp

```diff
@@ -92,6 +93,22 @@ chip::Platform::ScopedMemoryBuffer<uint8_t> sSsidBuf;
 chip::Platform::ScopedMemoryBuffer<uint8_t> sCredsBuf;
 chip::Platform::ScopedMemoryBuffer<uint8_t> sThreadBuf;
 chip::Controller::CommissioningParameters sCommissioningParameters;
+
+bool sTestCommissionerUsed = false;
```

Comment:
Suggest `g` prefix for global. It's more common, and this is effectively a global variable even if static to this module.

Reply:
I just used s here to match the style of the file already. When in Rome....